vembrane/__init__.py

```diff
@@ -58,44 +44,30 @@ def filter_vcf(
     annotation_keys = get_annotation_keys(header)
 
     for record in vcf:
-        # obtain annotation entries
-        ann = dict(
-            zip(
-                annotation_keys,
-                zip(
-                    *(
-                        parse_annotation_entry(entry)
-                        for entry in record.info.get("ANN", [])
-                    )
-                ),
+        # setup filter expression env
```

Comment:
We should keep building the env dict here with the global entries. This does not need to be redone in each iteration below. Below, we can just overwrite the ANN key.